src/components/ToastBar.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="toast__container" :class="{ error: toast.type === 'error' }">
```

Comment:
```suggestion
  <div class="toast__container" :class="toast.type === 'error' && error>
```

Also move toast.type === 'error' to a computed property